### DISCUSSION TITLE
Updated to 117.2.2.0 and getting Faulting Module libcef.dll on Windows 10 Education

### DISCUSSION THREAD

**Discussion by inetmarketeers** [cefsharp/CefSharp#4600] [General]
Has C++ 2019 installed. Not sure where to go for a solution

I opened Event Viewer Windows Logs\Application and found some APPCRASH events related to UploadBrowser.exe

Problem signature:
P1: UploadBrowser.exe
P2: 1.0.1.0
P3: 9fe7ab59
P4: libcef.dll
P5: 117.2.2.0
P6: 65137ca5
P7: 4000001f
P8: 0134ba8c

Faulting application name: UploadBrowser.exe, version: 1.0.1.0, time stamp: 0x9fe7ab59
Faulting module name: libcef.dll, version: 117.2.2.0, time stamp: 0x65137ca5
Exception code: 0x4000001f
Fault offset: 0x0134ba8c
Faulting process id: 0x5d10
Faulting application start time: 0x01d9f8a172364461
Faulting application path: C:\Program Files\Google Sites Factory 2\UploadBrowser.exe
Faulting module path: C:\Program Files\Google Sites Factory 2\libcef.dll
Report Id: c19780a6-b3d1-4420-9112-1703d416e566
Faulting package full name: 
Faulting package-relative application ID:

**Comment by amaitland** [comment]
What version are you upgrading from?

Are there any errors in the log file?

https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

If there's no errors in the log then you can enable crash reporting.

https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#crash-reporting
